src/interfaces/node.cpp

```diff
@@ -183,6 +183,17 @@ class NodeImpl : public Node
         LOCK(::cs_main);
         return ::ChainActive().Height();
     }
+
```

Comment:
In commit "Added best block hash to the NotifyHeaderTip and NotifyBlockTip signals." (4948d4ee50d9960b23f43e14938845b0bb21772b)

Should remove extra newlines between methods (here and below) for consistency with existing code

Reply:
done